src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,66 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import exceptions.DukeException;
+
+
+/** A class that functions as an abstraction of an event task. */
+public class Event extends Task {
+
+    public static final String WRONG_FORMAT_ERROR_STRING =
+            "Format for events: 'event [some event] /at [dd/mm/yyyy-hh:mm]'";
+    private static final DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("dd/MM/yyyy-HH:mm");
+    private static final DateTimeFormatter PRETTY_FORMAT = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+
+    private final Task.TaskType taskType = Task.TaskType.EVENT;
+    private final String taskName;
+    private final LocalDateTime eventTime;
```

Comment:
Since these variables are constants, shouldn’t they be in uppercase?
```suggestion
    private final Task.TaskType TASK_TYPE = Task.TaskType.EVENT;
    private final String TASK_NAME;
    private final LocalDateTime EVENT_TIME;
```